deepchem/models/tests/test_gcn.py

```diff
@@ -22,17 +22,16 @@
 def test_gcn_regression():
```

Comment:
Is this within the scope of this PR?

Reply:
That change was made by flake8 I believe. This commit was part of the Huber loss PR from where I made a new branch.

Reply:
Should be fine